pages/content/amp-dev/documentation/guides-and-tutorials/contribute/translations@pt_BR.md

```diff
@@ -0,0 +1,79 @@
+---
+$title: Contribua com traduções
+$order: 2
+description: "A missão do AMP inclui tornar a web um lugar mais acessível para todos, e não podemos atigir esse objetivo sem uma documentação localizada. A tradução da documentação AMP é um processo essencial que ajuda no sucesso e na saúde da comunidade AMP."
+
+formats:
+  - websites
+  - stories
+  - ads
+  - email
+---
```

Comment:
Can we remove everything from the frontmatter except for the `$title` and `$description`?

Reply:
Yes sure.